cprover_bindings/src/goto_program/expr.rs

```diff
@@ -293,6 +294,10 @@ impl Expr {
         &self.value
     }
 
+    pub fn size_of_annotation(&self) -> &Option<Type> {
```

Comment:
It's more idiomatic to return `Option<&Type>` as opposed to `&Option<Type>` (see https://users.rust-lang.org/t/what-is-the-preferred-way-to-return-option-fields/25100/5). You can use `as_ref` to do the conversion.

Reply:
Thank you!